netty/src/main/java/io/grpc/netty/AbstractNettyHandler.java

```diff
@@ -56,6 +58,9 @@
                        Http2Settings initialSettings) {
     super(decoder, encoder, initialSettings);
 
+    // Set the timeout for graceful shutdown.
+    gracefulShutdownTimeoutMillis(GRACEFUL_SHUTDOWN_TIMEOUT);
```

Comment:
What changed such that we are doing this in this PR? Wasn't `gracefulShutdownTimeoutMillis()` available in Beta6? Also, is something wrong with Netty's default of 30 seconds?